python/docs/source/index.rst

```diff
@@ -1,49 +1,67 @@
-
-Usage
------
-
-You can use the python wrapper directly like this:
-
-.. code-block:: python
-
-    from vowpalwabbit import pyvw
-    vw = pyvw.vw(quiet=True)
-    ex = vw.example('1 | a b c')
-    vw.learn(ex)
-    vw.predict(ex)
-
-Or you can use the included scikit-learn interface like this:
-
-.. code-block:: python
-
-    import numpy as np
-    from sklearn import datasets
-    from sklearn.model_selection import train_test_split
-    from vowpalwabbit.sklearn_vw import VWClassifier
-        # generate some data
-    X, y = datasets.make_hastie_10_2(n_samples=10000, random_state=1)
-    X = X.astype(np.float32)
-        # split train and test set
-    X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=256)
-        # build model
-    model = VWClassifier()
-    model.fit(X_train, y_train)
-        # predict model
-    y_pred = model.predict(X_test)
-        # evaluate model
-    model.score(X_train, y_train)
-    model.score(X_test, y_test)
+
+
+Why Vowpal Wabbit?
+------------------
+
+Vowpal Wabbit is a machine learning system which pushes the frontier of
+machine learning with techniques such as online, hashing, allreduce,
+reductions, learning2search, active, and interactive learning. There is
+a specific focus on reinforcement learning with several contextual
+bandit algorithms implemented and the online nature lending to the
+problem well. Vowpal Wabbit is a destination for implementing and
+maturing state of the art algorithms with performance in mind.
+
+-  **Input Format.** The input format for the learning algorithm is
+   substantially more flexible than might be expected. Examples can have
+   features consisting of free form text, which is interpreted in a
+   bag-of-words way. There can even be multiple sets of free form text
+   in different namespaces.
+-  **Speed.** The learning algorithm is fast – similar to the few other
+   online algorithm implementations out there. There are several
+   optimization algorithms available with the baseline being sparse
+   gradient descent (GD) on a loss function.
+-  **Scalability.** This is not the same as fast. Instead, the important
+   characteristic here is that the memory footprint of the program is
+   bounded independent of data. This means the training set is not
+   loaded into main memory before learning starts. In addition, the size
+   of the set of features is bounded independent of the amount of
+   training data using the hashing trick.
+-  **Feature Interaction.** Subsets of features can be internally paired
+   so that the algorithm is linear in the cross-product of the subsets.
+   This is useful for ranking problems. The alternative of explicitly
+   expanding the features before feeding them into the learning
+   algorithm can be both computation and space intensive, depending on
+   how it’s handled.
+
+`Visit the wiki to learn
+more. <https://github.com/VowpalWabbit/vowpal_wabbit/wiki>`__
+
+Getting Started
```

Comment:
I think this section is misleading, since it is talking primarily about the C++ not Python.

Reply:
then I can point to python(wiki) or will remove it.